src/components/test-slot/test-outcome/test-outcome.ts

```diff
@@ -8,7 +8,7 @@ import { TestStatus } from '../../../modules/tests/test-status/test-status.model
 import { StartE2EPracticeTest } from '../../../pages/fake-journal/fake-journal.actions';
 import { startsWith, isEmpty } from 'lodash';
 import { end2endPracticeSlotId } from '../../../shared/mocks/test-slot-ids.mock';
-import { JOURNAL_FORCE_CHECK_MODAL, CAT_B, CAT_BE, CAT_C } from '../../../pages/page-names.constants';
+import { JOURNAL_FORCE_CHECK_MODAL, CAT_B, CAT_BE, CAT_C, CAT_AMOD1 } from '../../../pages/page-names.constants';
```

Comment:
my PR will change the naming of this to CAT_A_Mod1, so this might get broken

Reply:
Thanks for the heads up